src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/Option.js

```diff
@@ -12,7 +11,6 @@ type Props<T> = {|
     anchorWidth: number,
     children: string,
     disabled: boolean,
-    focus: boolean,
```

Comment:
I have checked for this property, and it wasn't used in the entire code base. Therefore I've decided to remove it.

Reply:
Should we not focus always the selected element when opening the select dropdown?

Reply:
That's not done via this attribute (just double-checked, there is no occurence of this property, and the selects are still working fine).